package/CHANGELOG

```diff
@@ -27,6 +27,8 @@ Enhancements
   * Additional logger.info output when per-frame analysis starts (PR #3710)
   * Timestep has been converted to a Cython Extension type
     (CZI Performance track, PR #3683)
+  * Add higher performance AnalysisBase derived Major and Minor pair distances to
```

Comment:
We usually go newest first for these entries, could you move it to the top of the enhancements list?